lib/browsers/phantomjs-control-script.js

```diff
@@ -66,7 +66,8 @@ page.onInitialized = function (arg1) {
                 window.callPhantom({
                     name: 'slaveDispose'
                 });
-            }
+            },
+            replaceConsole: true
```

Comment:
@benouat Thank you for your review! I have renamed the variable as you suggested.